core/Object.php

```diff
@@ -908,10 +908,12 @@ protected function removeMethodsFrom($property, $index = null) {
 		$methods = $this->findMethodsFromExtension($extension);
 		if ($methods) {
 			foreach ($methods as $method) {
-				$methodInfo = self::$extra_methods[$this->class][$method];
+				if (isset(self::$extra_methods[$this->class][$method])) {
```

Comment:
How about doing an early `continue` instead of indenting the contents of this statement?